src/common/CMakeLists.txt

```diff
@@ -1,8 +1,22 @@
+
+set (MONGOC_ENABLE_TESTING 0)
```

Comment:
Can you do a global find/replace to change `MONGOC_ENABLE_TESTING` with `MONGOC_ENABLE_DEBUG_ASSERTIONS`?